v0/leanplum/transform.js

```diff
@@ -45,11 +45,6 @@ function setValues(payload, message, mappingJson) {
 
 function constructPayload(message, name, destination) {
   mappingJson = mappingConfig[name];
-  let rawPayload = {
```

Comment:
is this not required?

Reply:
I think this file changed by mistake I will add the deleted part.